SquareLine/boards/custom_waveshare_7inch/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name":"Custom WaveShare 7inch",
+    "version":"1.0.0",
+    "mcu":"ESP32S3",
+
+    "screen_width":"800",
+    "screen_height":"480",
+    "screen_color_swap":true,
+
+    "short_description":"WaveShare 7inch Display",
+    "long_description":"Example of the custom BSP and custom LCD",
+
+    "placeholders":
+    {
+        "__ESP_BOARD_INCLUDE__": "bsp/ws_7inch.h",
+        "__ESP_BOARD_I2C_INIT__": "/* Initialize I2C (for touch and audio) */\n    bsp_i2c_init();"
```

Comment:
Let's consider this usecase:

User has this specific display on his custom board. He needs to change I2C pinout. How can he do it?

Reply:
I think, it is ok in component header file.